types/entity.d.ts

```diff
@@ -0,0 +1,42 @@
+/// <reference types="node" />
+/// <reference types="vec3" />
+/// <reference types="prismarine-item" />
+
+import {EventEmitter} from 'events';
+import { Vec3 } from 'vec3';
+import { Item } from 'prismarine-item';
+import { ChatMessage } from 'prismarine-chat';
+
+
+export class Entity extends EventEmitter {
```

Comment:
is there any reason you brought this to it's own file rather than just leaving it in the index.d.ts?